Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:16.04
 MAINTAINER Adrien Leravat <Pixep@users.noreply.github.com>
 
 # Install prerequisites
-RUN echo nameserver 8.8.8.8 > /etc/resolv.conf
+CMD echo nameserver 8.8.8.8 >> /etc/resolv.conf
```

Comment:
Could you explain quickly the reasoning for the `>>`? I'm not sure how its going to play with the default `resolv.conf` content.